### ISSUE TITLE
基建换班菲亚梅塔识别错误

### ISSUE THREAD

**Issue by luwh2000** [MaaAssistantArknights/MaaAssistantArknights#8386]
### 在提问之前...

- [X] 我理解 Issue 是用于反馈和解决问题的，而非吐槽评论区，将尽可能提供更多信息帮助问题解决
- [X] 我填写了简短且清晰明确的标题，以便开发者在翻阅 issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等
- [X] 我已查看最新测试版本的更新内容，并未提及该 bug 已被修复的情况
- [X] 我已检查了置顶议题（pinned issue），确认我的问题未被提及

### 问题描述

基建换班MAA识别不到菲亚梅塔。在asst.log里面有奇怪的“菲菲亚梅塔”的字段，但我的custom_infrast里填写正常。

### Version

Resource Version: 千秋一粟
UI Version: v5.1.1-alpha.1.d030.g078feead6
Core Version: v5.1.1-alpha.1.d030.g078feead6

### 日志文件

[gui.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/14377752/gui.log)
[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/14377753/asst.log)
时间大致为 2024-02-22 12:56
[1703662148853537.json](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/14377798/1703662148853537.json)
这是自定义基建换班表

**Comment by Constrat** [comment]
This instance at 12:40:
```
[2024-02-22 12:40:03.201][TRC][Px6c0c][Tx0cc4] asst::WordOcr [{ 菲亚梅塔: [ 0, 0, 82, 19 ], score: 0.999424 }] by OCR Rec , cost 18 ms
[2024-02-22 12:40:03.201][TRC][Px6c0c][Tx0cc4] Proceed [{ 菲菲亚梅塔: [ 39, 1, 82, 19 ], score: 0.999424 }]
```
```json
".*亚梅塔",
"菲亚梅塔"
```

**Comment by Constrat** [comment]
My bad, I didn't notice the different tags in the commits and the version mentioned. 
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/ebfb0790-d456-4253-9e5d-71e0491d0ee1)
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/edc382fd-4261-45a2-af86-729eeabc3d70)
Makes little to no sense showing the latest tag and hiding the older ones... Sorry for the troubles.
@luwh2000 This has already been fixed in version `alpha/v5.1.1-alpha.1.d051.g573a16e14` with 6f00c2db39bf55073b0335fbceee57d7e5821585
